### ISSUE TITLE
Way too much logging when there is no live version of content

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#17551]
I am testing the SPA and I am seeing a ton of this in the logs.  I think this should be debug?
```
17:44:27.149  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier af5d2a9d-86ed-421d-a9e6-5e5dae0e484a
17:44:27.149  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier af5d2a9d-86ed-421d-a9e6-5e5dae0e484a
17:44:27.149  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier af5d2a9d-86ed-421d-a9e6-5e5dae0e484a
17:44:27.151  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.151  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.151  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.152  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.152  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.152  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.153  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.153  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.153  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.153  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.153  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.153  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.154  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.154  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.155  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.155  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.155  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.155  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.156  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.156  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.156  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 8ccfa397-4369-44bb-b450-33151387eb02
17:44:27.156  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.156  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
17:44:27.156  WARN  business.ESContentletAPIImpl - Error trying to get a live version for content with identifier 4694d40b-d9be-4e09-b031-64ee3e7c9642
```

**Comment by erickgonzalez** [comment]
Now instead of logging it several times, only one it's logged like this message:
`WARN  business.ESContentletAPIImpl - No live version for contentlet identifier 83c55f5b-e498-438c-884a-5e73824e205e (every 5000 millis)`

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/17566

**Comment by bryanboza** [comment]
Fixed, tested on release 5.2.1 // Postgres // FF

**Comment by nollymar** [comment]
Tested on master. Working as expected.
